assets/src/adaptivity/rules-engine.ts

```diff
@@ -261,7 +261,7 @@ export const defaultWrongRule = {
   },
 };
 
-export const findReferencedActivitiesInConditions = (conditions: any) => {
+export const findReferencedActivitiesInConditions = (conditions: any, actions: any) => {
```

Comment:
very sorry :D I was thinking that the actions were already going to be a part of the condition item; I do think that it would be good to have separate distinct methods for doing this, esp. since you had to edit the locations anyway